vendor/istio.io/api/mesh/v1alpha1/config.proto

```diff
@@ -202,46 +210,13 @@ message MeshConfig {
   // and pass to sds server, which will be used to request key/cert eventually.
   // If both enable_sds_token_mount and sds_use_k8s_sa_jwt are set to true, enable_sds_token_mount(trustworthy jwt) takes precedence.
   // This isn't supported for non-k8s case.
-  bool sds_use_k8s_sa_jwt = 30;
+  bool sds_use_k8s_sa_jwt = 29;
```

Comment:
This looks like backward incompatible change cc @quanjielin

Reply:
I see a bunch of changes in vendor that look strange, shouldn't we update vendor separately?

Reply:
cc @rshriram - this was removed in https://github.com/istio/api/pull/742.